docs/openqaoa-jobs/braket-jobs.md

```diff
@@ -0,0 +1,28 @@
+# Bryong your own OpenQAOA container to Amazon Braket Hybrid Jobs
+
+If you are wondering what is an Amazon Braket Hybrid Jobs, check out Braket's very answer
+
+> With Hybrid Jobs, you can run algorithms that use both classical and quantum compute resources. These hybrid quantum-classical algorithms can help you optimize the performance of the quantum devices currently available. These algorithms are designed to maximize the benefits and reduce the drawbacks of both types of computational systems and are generally used to find the ground state or global minimum of a particular system.
```

Comment:
We could add quotes and specify the link to braket's doc that defines this for completeness

Reply:
Yes, I forgot to add it!